Modules/Product/Http/Services/ProductService.php

```diff
@@ -235,57 +217,57 @@ public function arrangeProduct($request): array
             ->orderBy('order_by', 'ASC')
             ->get();
 
-        foreach ($products as $v) {
-            $productVariations = Product::find($v->id)->productVariation()->where('status', '1')->get();
+        foreach ($products as $product) {
+            $productVariations = Product::find($product->id)->productVariations()->where('status', '1')->get();
             $productVariationsCount = $productVariations->count();
             if ($productVariationsCount > 0) {
-                $variantMinPrice = Product::find($v->id)->productVariation()->min('price');
+                $variantMinPrice = Product::find($product->id)->productVariations()->min('price');
                 $price = $variantMinPrice . '+';
-                $variantStock = Product::find($v->id)->productVariation()->sum('price');
+                $variantStock = Product::find($product->id)->productVariations()->sum('price');
```

Comment:
Instead of querying to Product model, query on relation which you'll get using with().